src/Workspaces/CSharp/Portable/Simplification/Reducers/AbstractCSharpReducer.AbstractReductionRewriter.cs

```diff
@@ -153,19 +157,6 @@ private static SyntaxNode GetParentNode(CrefSyntax cref)
                 return node;
             }
 
-            protected SyntaxNode SimplifyExpression<TExpression>(
-                TExpression expression,
-                SyntaxNode newNode,
-                Func<TExpression, SemanticModel, CSharpSimplifierOptions, CancellationToken, SyntaxNode> simplifier)
-                where TExpression : SyntaxNode
-            {
-                var parentNode = GetParentNode(expression);
-                if (parentNode == null)
-                    return newNode;
-
-                return SimplifyNode(expression, newNode, parentNode, simplifier);
-            }
```

Comment:
got rid of this helper.  everyone calls through the same SimplifyNode function.  Another bug was this was being bypassed in the 'default expression simplifier'.